pkg/manifests/types.go

```diff
@@ -286,6 +289,14 @@ type NodeExporterCollectorTcpStatConfig struct {
 	Enabled bool `json:"enabled,omitempty"`
 }
 
+// The `NodeExporterCollectorNetDevConfig` resource works as an on/off switch for
+// the `netdev` collector of the `node-exporter` agent.
+// By default, the `netdev` collector is enabled.
+type NodeExporterCollectorNetDevConfig struct {
+	// A Boolean flag that enables or disables the `netdev` colletor.
```

Comment:
We need to document clearly which metrics will be missing hwen the collector is disabled.